### ISSUE TITLE
No-IP support.

### ISSUE THREAD

**Issue by Proxyl3ss** [NYAN-x-CAT/AsyncRAT-C-Sharp#116]
Would it be easy for you to add support for like a DDNS service so I do not lose all my connections when my public IP changes?

~Thanks in advance,
Proxyless

**Comment by Abraham313** [comment]
No-IP is a stupid they keep all of your logs.You should not use no-ip.If you want then just put your dns into dns connection option from builder and update your IP from client or web browser.It's OK you never lose your connection even IP changes.

**Comment by Proxyl3ss** [comment]
I know but I haven't found any other way of not losing my connections. Any
suggestions?

**Comment by NYAN-x-CAT** [comment]
DNS is supported already. but it's better to use the Pastebin feature.
Just create new account in pastebin and create a new paste

should be something like this:
```
https://pastebin.com/raw/s14cUU5G
```
